src/modules/decentralized-web-node/entities/message.entity.ts

```diff
@@ -0,0 +1,52 @@
+import { Column, Entity, PrimaryColumn } from 'typeorm';
+import { CollectionsQueryDescriptorDto } from '../dtos/collections-query-descriptor.dto';
+import { CollectionsWriteDescriptorDto } from '../dtos/collections-write-descriptor.dto';
+import { MessageDto } from '../dtos/message.dto';
+
+@Entity()
+export class Message {
+  static create(data: Partial<Message>): Message {
+    const entity = new Message();
+    Object.assign(entity, data);
+    return entity;
+  }
+
+  @PrimaryColumn()
+  objectId: string;
+
+  @Column()
+  createdBy: string;
```

Comment:
Is `createdBy` a part of the spec? I think it would be better to use `target` here?
The spec doesn't really specify `target` in text, but it's in all of the examples: https://identity.foundation/decentralized-web-node/spec/#messages
```suggestion
  target: string;
```